src/lib/whatInput.ts

```diff
@@ -0,0 +1,427 @@
+// Taken from https://github.com/ten1seven/what-input/blob/master/src/scripts/what-input.js
```

Comment:
If we are copying the whole library, I think might be worth to optimize it a bit, and get rid of things that we are not using. For example, I am not sure about ```data-whatintent```, as we are reading value from ```data-whatinput```.  Also additional props like ```ignoreKeys``` and so on are not used and can be removed. 

Basically, what we need it's just to get info if the last input was 'keyboard'. So I suggest to leave everything that is related to it and remove anything else. 
``` touchstart``` and ```touchend``` are triggered only for mobile, so we might not need it also as input won't be changed to 'keyboard' on mobile

Reply:
Agree, but this will mean that we want to adopt it, but I'm not sure that actually need to do this.

Reply:
We also don't need `fireFunctions` and many other stuff, but as I see: we need to fix our bug and just wait for this function in the module.

And may be, a more better solution that matches React's ideas. Because even with `setInput()` it will still be a monkey patch.

Reply:
we can start with removing unused things (because it's easy and fast :)) as waiting for this thing to be fixed in a module takes an unknown time.
Reinventing something new, more in React way, might be considered for some future RFC, but at least we need to get back from authors that they won't fix it.

Reply:
Please understand my position there, we can two options.

Option 1. We want to get an enchantment in the package and we will temporary copy its code as is. We know that this code is stable and working, we should not touch it at all.
Option 2. We fork the package and adopt it for our needs. In this case I of course will remove all unused options.

Reply:
Option 3: Let's try deleting as much of this as possible.  The `isFromKeyboard` abstraction currently has lots of code that is unnecessary. I'm also not sure if we need this utility or if it can be done in a more simple way.

Reply:
I reworked this as you suggested guys.

- drop `what-input`
- removed `isFromKeyboard.ts` with util
- `typescriptUtils.ts` also removed

For now, we have only a `isFromKeyboard()` function. Issue with events priority was resolved by `useCapture`.